README.md

```diff
@@ -183,17 +192,15 @@ Spectacle comes with a built in presenter mode. It shows you a slide lookahead,
 
 ![http://i.imgur.com/jW8uMYY.png](http://i.imgur.com/jW8uMYY.png)
 
-Otherwise, it can also show you a stopwatch to count the elapsed time:
+You also have the option of a stopwatch to count the elapsed time:
 
 ![http://i.imgur.com/VDltgmZ.png](http://i.imgur.com/VDltgmZ.png)
 
 To present:
 
-- Run `npm start`
-- Open two browser windows on two different screens
-- On your screen visit [http://localhost:3000/](http://localhost:3000/). You will be redirected to a URL containing the slide id.
-- Add `presenter&` or `presenter&timer` immediately after the questionmark, e.g.: [http://localhost:3000/#/0?presenter](http://localhost:3000/#/0?presenter) or [http://localhost:3000/#/0?presenter&timer](http://localhost:3000/#/?presenter&timer&_k=wbyhif)
-- On the presentation screen visit [http://localhost:3000/](http://localhost:3000/)
+- Run `npm start`. You will be redirected to a URL containing your presentation or visit [http://localhost:3000/#/](http://localhost:3000/#/)
+- Open a second browser window on a different screen
+- Add `0?presenter` or `0?presenter&timer` immediately after the `/`, e.g.: [http://localhost:3000/#/0?presenter](http://localhost:3000/#/0?presenter) or [http://localhost:3000/#/0?presenter&timer](http://localhost:3000/#/?presenter&timer&_k=wbyhif)
```

Comment:
We might want to remove the index from this query parameter to avoid confusion. The user only needs to append their url with `?`, the index before it can vary.